intel_extension_for_transformers/neural_chat/examples/finetuning/instruction/README.md

```diff
@@ -402,13 +402,40 @@ mpirun -f nodefile -n 16 -ppn 4 -genv OMP_NUM_THREADS=56 python3 finetune_clm.py
     --dataset_concatenation \
     --do_train \
     --trust_remote_code True \
+    --use_fast_tokenizer True \
     --tokenizer_name "EleutherAI/gpt-neox-20b" \
     --no_cuda \
     --ddp_backend ccl \
 ```
 you could also indicate `--peft` to switch peft method in P-tuning, Prefix tuning, Prompt tuning, LLama Adapter, LORA,
 see https://github.com/huggingface/peft
 
+## 3. Multi-node Fine-tuning in AWS m7i SPR instances
+
+### Build Docker image with customized SSH server port from scratch
+AWS instances have a SSH server on by default, so we need to start SSH Server with different port inside the docker instance.  
+Users could pick their CUSTOM_PORT but we should not use 22 as the SSH Server port inside the docker instance.  
+Please refer to this section : [How to build docker images for NeuralChat FineTuning](../../../docker/finetuning/README.md#21-build-docker-image) and add `--build-arg SSHD_PORT=<CUSTOM_PORT>` to build docker image from scratch.   
+
+ex : using 2345 as the CUSTOM_PORT  
+```bash
+docker build --build-arg UBUNTU_VER=22.04 --build-arg SSHD_PORT=2345 -f intel-extension-for-transformers/intel_extension_for_transformers/neural_chat/docker/Dockerfile -t ${IMAGE_NAME}:${IMAGE_TAG} . --target cpu
+```  
+
+### Add one AWS inbound rule for distributed training
+Allow all network traffic inside the cluster, so that distributed training runs unencumbered.   
+AWS provides a safe and convenient way to do this with security groups. We just need to create a security group that allows all traffic from instances configured with that same security group and make sure to attach it to all instances in the cluster.  
+Here's how my setup looks.  
+<img width="655" alt="image" src="https://github.com/intel-ai-tce/intel-extension-for-transformers/assets/21761437/baf98fdd-14ff-457c-afed-a387b164c2df">  
```

Comment:
better to add the assets under itrex https://github.com/intel/intel-extension-for-transformers/tree/main/intel_extension_for_transformers/neural_chat/assets

Reply:
addressed it accordingly.